lighthouse-core/test/report/html/renderer/performance-category-renderer-test.js

```diff
@@ -140,7 +140,8 @@ describe('PerfCategoryRenderer', () => {
     const diagnosticSection = categoryDOM.querySelectorAll('.lh-category > .lh-audit-group')[2];
 
     const diagnosticAudits = category.auditRefs.filter(audit => audit.group === 'diagnostics' &&
-        audit.result.score !== 1 && audit.result.scoreDisplayMode !== 'not-applicable');
+        !Util.showAsPassed(audit.result) &&
+        audit.result.scoreDisplayMode !== 'not-applicable');
```

Comment:
oh, sorry, should have put the comment below this line. `showAsPassed()` includes the `not-applicable` check :)

Reply:
That's my bad; I should have noticed the check showAsPassed. :+1: Fixed in latest commit.